app/workers/vbms/submit_dependents_pdf_job.rb

```diff
@@ -36,7 +36,7 @@ def send_error_to_sentry(error, saved_claim_id)
 
     def generate_pdf(claim, submittable_686, submittable_674)
       claim.upload_pdf('686C-674') if submittable_686
-      claim.upload_pdf('21-674') if submittable_674
+      claim.upload_pdf('21-674', doc_type: '143') if submittable_674
```

Comment:
Just want to make sure that the 143 doc type above isn't a typo. I noticed that the other changes have a doc type of 148

Reply:
@LindseySaari thank you for double checking.  It is not a typo.  148 is the doc type that should be used as the "default".  In this one particular line, we are generating a different doc type, so we do want 143 here.  But again, thank you for checking as that could easily have been a typo!

Reply:
Ok great!